test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/Parsers/FunctionParameterParserTests.cs

```diff
@@ -21,6 +21,19 @@ namespace Microsoft.OData.Tests.UriParser.Parsers
     /// </summary>
     public class FunctionParameterParserTests
     {
+        [Theory]
+        [InlineData("1", "1", null)]

```

Comment:
Is this a scenario where the function is being called with one parameter without specifying the name?